src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java

```diff
@@ -44,4 +54,19 @@ public Flight deleteFlight(Long id) {
     public List<Flight> getByFromToDate(String departureAirport, String arrivalAirport, LocalDateTime departureDateFrom, LocalDateTime departureDateTo) {
         return flightDAO.getByFromToDate(departureAirport, arrivalAirport, departureDateFrom, departureDateTo);
     }
+
+    private long updateFlightPrice(Flight flight) {
+        LocalDateTime dateAfter = flight.getDepartureDate();
+        LocalDateTime dateBefore = LocalDateTime.now();
+        long daysBetween = DAYS.between(dateBefore, dateAfter);
+        long minPrice = flight.getInitialPrice();
+        long price = (long) (minPrice + (60 - daysBetween) * (60 - daysBetween) * (minPrice / 3600.0));
+        System.out.println(daysBetween);
```

Comment:
...

Reply:
nobody saw it... i removed it before your comment